<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTableWidget" name="tableWidget"/>
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>project</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>run</string>
    </property>
    <addaction name="actionrun_stop"/>
    <addaction name="actiondataflow"/>
    <addaction name="actionclear"/>
    <addaction name="separator"/>
    <addaction name="actionup"/>
    <addaction name="actiondown"/>
    <addaction name="separator"/>
    <addaction name="actiontop"/>
    <addaction name="actionend"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionrun_stop">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/运行.png</normaloff>:/运行.png</iconset>
   </property>
   <property name="text">
    <string>run &amp; stop</string>
   </property>
  </action>
  <action name="actionclear">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/清除.png</normaloff>:/清除.png</iconset>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </action>
  <action name="actionup">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/向上1.png</normaloff>:/向上1.png</iconset>
   </property>
   <property name="text">
    <string>up</string>
   </property>
  </action>
  <action name="actiondown">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/向下1.png</normaloff>:/向下1.png</iconset>
   </property>
   <property name="text">
    <string>down</string>
   </property>
  </action>
  <action name="actiontop">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/顶部.png</normaloff>:/顶部.png</iconset>
   </property>
   <property name="text">
    <string>top</string>
   </property>
  </action>
  <action name="actionend">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/底部.png</normaloff>:/底部.png</iconset>
   </property>
   <property name="text">
    <string>end</string>
   </property>
  </action>
  <action name="actiondataflow">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/流量监控.png</normaloff>:/流量监控.png</iconset>
   </property>
   <property name="text">
    <string>dataflow</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
